packages/app-mobile/components/accessibility/FocusControl/FocusControl.tsx

```diff
@@ -0,0 +1,11 @@
+import FocusControlProvider from './FocusControlProvider';
+import MainAppContent from './MainAppContent';
+import ModalWrapper from './ModalWrapper';
+
+const FocusControl = {
+	Provider: FocusControlProvider,
+	ModalWrapper,
+	MainAppContent,
+};
+
+export default FocusControl;
```

Comment:
The goal of wrapping these components in a `FocusControl` object was to make it clear to a reader that `MainAppContent` (and similar components) are specific to the `FocusControl` logic. An alternative could be to rename `ModalWrapper` to `FocusControlModalWrapper` and `MainAppContent` to `FocusControlMainAppContent` (or similar).